webapp/api_handlers.go

```diff
@@ -374,3 +377,122 @@ func handleAPIDiffPost(w http.ResponseWriter, r *http.Request) {
 	}
 	w.Write(bytes)
 }
+
+func apiManifestHandler(w http.ResponseWriter, r *http.Request) {
+	sha, err := ParseSHAParamFull(r)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+	ctx := appengine.NewContext(r)
+	if manifest, err := getManifestForSHA(ctx, sha); err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
+	} else if manifest == nil {
```

Comment:
Letting both `manifest` and `err` be nil seems like an odd API, should `getManifestForSHA` return an error at its final line instead?

Reply:
Done.